### PR TITLE
feat(ApplicationFlags): add application_auto_moderation_rule_create_badge

### PR DESCRIPTION
**Pull request by JustaSqu1d**
## Summary

https://github.com/discord/discord-api-docs/pull/6021

## Information

- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [x] I have updated the documentation to reflect the changes.
- [x] If `type: ignore` comments were used, a comment is also left explaining why.
- [x] I have updated the changelog to include these changes.

### CODE REVIEW

**Review by Lulalaby — APPROVED** [review]

**Review by ChickenDevs — APPROVED** [review]

File: discord/flags.py [hunk 1]
```diff
@@ -1338,6 +1338,11 @@ def group_dm_create(self):
         """:class:`bool`: Returns ``True`` if the application can create group DMs."""
         return 1 << 5
 
+    @flag_value
+    def application_auto_moderation_rule_create_badge(self):
+        """:class:`bool`: Returns ``True`` if the application uses the Auto Moderation API"""
```
Comment (plun1331) [inline discord/flags.py:1343]:
```suggestion
        """:class:`bool`: Returns ``True`` if the application uses the Auto Moderation API."""
```

**Review by plun1331 — CHANGES_REQUESTED** [review]

**Comment by Lulalaby** [comment]
mhh do we need a version added 🤣

**Comment by plun1331** [comment]
and now we wait for tests to run again

**Comment by plun1331** [comment]
> mhh do we need a version added 🤣

maybe

File: discord/flags.py [hunk 2]
```diff
@@ -1338,6 +1338,14 @@ def group_dm_create(self):
         """:class:`bool`: Returns ``True`` if the application can create group DMs."""
         return 1 << 5
 
+    @flag_value
+    def application_auto_moderation_rule_create_badge(self):
+        """:class:`bool`: Returns ``True`` if the application uses the Auto Moderation API
```
Comment (plun1331) [inline discord/flags.py:1343]: you got rid of the period again
```suggestion
        """:class:`bool`: Returns ``True`` if the application uses the Auto Moderation API.
```

**Review by plun1331 — APPROVED** [review]

**Comment by Lulalaby** [comment]
your own approval won't work

**Comment by plun1331** [comment]
> your own approval won't work

god damn it

**Review by Lulalaby — APPROVED** [review]

**Review by ChickenDevs — APPROVED** [review]

**Pull request merged**